keystrokes/src/components/Character/CharacterList.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react';
+import '../../stylesheets/character.css';
+import styleGuide from '../../stylesheets/style';
+import { ref } from '../../config/constants';
+import { List, ListItem, Typography, Divider } from '@material-ui/core';
+import CharacterCreationModal from './CharacterCreationModal';
+
+function CharacterList(props) {
+	const [characters, setCharacters] = useState({});
+	const [currentCharacter, setCurrentCharacter] = useState()
```

Comment:
Would it make sense to have a default value of null for the currentCharacter?

Reply:
ah i did this so i could do `currentCharacter && ... ` but i had goofed up a test of an empty string, so turns out (as you said) i can just do `!!currentCharacter &&`  instead!